components/donnees-nationales/re-users.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types'
+import ReUser from '@/components/donnees-nationales/re-user'
+import {orderBy} from 'lodash'
+
+import colors from '@/styles/colors'
+
+function ReUsers({data}) {
+  const items = orderBy(data, 'name', 'asc')
+
+  return (
+    <>
+      <div className='re-users-container'>
+        {items.map(item => <ReUser key={item.name} reUserInfos={item} />)}
```

Comment:
Tu pourrais déstructurer l'utilisateur pour utiliser les valeurs qui t'intéressent dans le composant.

```suggestion
        {items.map(user => <ReUser key={item.name} {...user} />)}
```